Converted CSS to Sass/SCSS

bubbleheadinc opened the pull request:
Converted CSS to Sass/SCSS for easier import into Sass projects, abstraction for easier theming, configuration for usage variation. Cleaned up repetitive/loose styles. Moved body styles to demo index styles. Configurable sass to turn on/off horiztontal/vertical/accordion styles.

bubbleheadinc committed:
Moved all styles to sass/scss structure to allow for easier modification/customization. Cleaned up repetitive/loose styles. Moved body styles to demo index styles. Configurable sass to turn on/off horiztontal/vertical/accordion styles.

samsono commented:
Hi,

Thanks for you efforts toward the plugin development I really appriciate.

As of now I'm going thorough your code and will update you about the marge.

On Fri, Jul 12, 2013 at 12:25 AM, bubbleheadinc notifications@github.comwrote:

> Converted CSS to Sass/SCSS for easier import into Sass projects,
> abstraction for easier theming, configuration for usage variation. Cleaned
> up repetitive/loose styles. Moved body styles to demo index styles.
> 
> ## Configurable sass to turn on/off horiztontal/vertical/accordion styles.
> 
> You can merge this Pull Request by running
> 
>   git pull https://github.com/bubbleheadinc/Easy-Responsive-Tabs-to-Accordion master
> 
> Or view, comment on, or merge it at:
> 
>   https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8
> Commit Summary
> - Moved all styles to sass/scss structure to allow for easier
>   modification/customization. Cleaned up repetitive/loose styles. Moved body
>   styles to demo index styles. Configurable sass to turn on/off
>   horiztontal/vertical/accordion styles.
> 
> File Changes
> - _M_ Index.htmlhttps://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-0(34)
> - _M_ css/easy-responsive-tabs.csshttps://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-1(283)
> - _A_ sass/_accordion-styles.scsshttps://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-2(74)
> - _A_ sass/_base.scsshttps://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-3(40)
> - _A_ sass/_horizontal-tabs.scsshttps://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-4(28)
> - _A_ sass/_utility.scsshttps://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-5(15)
> - _A_ sass/_variables.scsshttps://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-6(38)
> - _A_ sass/_vertical-styles.scsshttps://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-7(63)
> - _A_ sass/easy-responsive-tabs.scsshttps://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-8(29)
> 
> Patch Links:
> 
>    -
>    https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8.patch
>    -
>    https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8.diff

## 

Thanks & Regards,
Samson
9177852826.

bubbleheadinc commented:
Hi Samson...

Sounds good. Let me know if there is anything I need to modify.

Thanks,
Sam

On Tue, Jul 16, 2013 at 2:59 AM, Samson Onna notifications@github.comwrote:

> Hi,
> 
> Thanks for you efforts toward the plugin development I really appriciate.
> 
> As of now I'm going thorough your code and will update you about the marge.
> 
> On Fri, Jul 12, 2013 at 12:25 AM, bubbleheadinc <notifications@github.com
> 
> > wrote:
> > 
> > Converted CSS to Sass/SCSS for easier import into Sass projects,
> > abstraction for easier theming, configuration for usage variation.
> > Cleaned
> > up repetitive/loose styles. Moved body styles to demo index styles.
> > 
> > ## Configurable sass to turn on/off horiztontal/vertical/accordion styles.
> > 
> > You can merge this Pull Request by running
> > 
> > git pull
> > https://github.com/bubbleheadinc/Easy-Responsive-Tabs-to-Accordion master
> > 
> > Or view, comment on, or merge it at:
> > 
> > https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8
> > Commit Summary
> > - Moved all styles to sass/scss structure to allow for easier
> >   modification/customization. Cleaned up repetitive/loose styles. Moved
> >   body
> >   styles to demo index styles. Configurable sass to turn on/off
> >   horiztontal/vertical/accordion styles.
> > 
> > File Changes
> > - _M_ Index.html<
> >   https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-0
> >   (34)
> > - _M_ css/easy-responsive-tabs.css<
> >   https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-1
> >   (283)
> > - _A_ sass/_accordion-styles.scss<
> >   https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-2
> >   (74)
> > - _A_ sass/_base.scss<
> >   https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-3
> >   (40)
> > - _A_ sass/_horizontal-tabs.scss<
> >   https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-4
> >   (28)
> > - _A_ sass/_utility.scss<
> >   https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-5
> >   (15)
> > - _A_ sass/_variables.scss<
> >   https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-6
> >   (38)
> > - _A_ sass/_vertical-styles.scss<
> >   https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-7
> >   (63)
> > - _A_ sass/easy-responsive-tabs.scss<
> >   https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8/files#diff-8
> >   (29)
> > 
> > Patch Links:
> > 
> > ## 
> > 
> > ## https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8.patch
> > 
> > https://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8.diff
> 
> ## 
> 
> Thanks & Regards,
> Samson
> 9177852826.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/samsono/Easy-Responsive-Tabs-to-Accordion/pull/8#issuecomment-21024673
> .

## 

_Samuel Vacca_
Point Dynamics, LLC
svacca@pointdynamics.com
P: 678-951-0712 ext. 204
F: 678-819-5204
www.pointdynamics.com
@pointdynamics http://twitter.com/#!/pointdynamics

samsono commented:
Hey, I apologies, if possible can u please updated the SASS code
